index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
+  <link rel="stylesheet"
+  href="https://fonts.googleapis.com/css?family=Cormorant">
+</head>
+<body>
+  <header>
+    <h1 class="title">M. Chau</h1>
+    <nav>
+      <a href=".">About</a>
```

Comment:
I don't have another place to put this, but while I think it's perfectly fine to change the file structure here, your repro does still have the orphaned about.html file. :3